### ISSUE TITLE
How to proxy with transfer-encoding chunked?

### ISSUE THREAD

**Issue by Mousaka** [SuaveIO/suave#750]
I'm trying to proxy to an endpoint using [proxy](https://github.com/SuaveIO/suave/blob/master/src/Suave/Proxy.fs) from a Suave server where the remote endpoint is giving me a response with `transfer-encoding: chunked`.

My Suave server logs 
```
[16:33:33 INF] POST Response (StatusCode 200) at /graphql took 8 ms
[16:34:17 INF] POST Request at /graphql
```
But my client that made the call says
```
curl: (56) Illegal or missing hexadecimal sequence in chunked-encoding
```

My question:
Does proxy not work with transfer-encoding chunked data?

**Comment by ademar** [comment]
No, the current implementation does not handle chunked data properly. 

It needs a few adjustments.
